.github/workflows/build-and-runtime-test.yml

```diff
@@ -133,3 +133,19 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_METRIC_LOGGER }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_METRIC_LOGGER }}
           AWS_REGION: us-east-2
+
+  upload-files-on-error:
+    #send video and screenshots on failure as an artifact to be checked later
+    runs-on: ubuntu-latest
+    environment: ci
```

Comment:
Oops, one more nit:
```suggestion
```

this shouldn't be necessary because we aren't using any secrets from it